odl/operator/default_ops.py

```diff
@@ -220,7 +198,7 @@ def __init__(self, space, a, b):
         self.a = a
         self.b = b
 
-    def _apply(self, x, out):
+    def _call(self, x, out):
```

Comment:
Sloppy of me. Should be optional here, too. Actually, our internal operators should IMO implement the optional pattern whenever it makes sense and not rely on the default in-place or out-of-place.